### DISCUSSION TITLE
v3.2

### DISCUSSION THREAD

**Discussion by kaustubhgupta** [kaustubhgupta/PortfolioFy#69] [General]
This release covers big updates in the project:

**TL;DR: 5 New GitHub stats added, new projects sort, documentation revamped, code restructured, improved contribution guide files and fixed a lot of bugs.**

- #36 GitHub Stats to be integrated, PR #61 
- #38 Documentation revamped, PR #65, #66, #67, #68 
- #40 HTTPS updation, PR #41 
- #45 Allow footer not working, PR #46 
- #47 Updation Time, PR #48 
- #49 Sort by forks, PR #54 
- #50 Restructure the code, PR #51 
- #52 Add local testing file, PR #53 
- #56 strtobool is not working as expected, eval not working for bash type shells, PR #57 
- #60 Committer commits only for "master branch" for new files. PR #61 
- Miscellaneous Issues PR: #64, #63, #62

<hr /><em>This discussion was created from the release <a href='https://github.com/kaustubhgupta/PortfolioFy/releases/tag/v3.2'>v3.2</a>.</em>

**Comment by kaustubhgupta** [comment]
@MRDGH2821 what do you think about these changes? 😄

**Reply by MRDGH2821** [reply]
Woah, that's a lot of changes in backend & frontend😶

**Reply by MRDGH2821** [reply]
Looks nice!

**Reply by kaustubhgupta** [reply]
Thanks for the feedback!
